collections_hierarchy/main_collections.py

```diff
@@ -11,7 +11,7 @@ class List(ComparableMixin,
     DATA_DEFAULT_INITIAL = []
 
     def get_elements(self):
-        pass
+        return [x for x in getattr(self, self.DATA_ATTR_NAME)]
 
```

Comment:
We had some hints for this, but I think creating a loop wasn't necessary in this instance.  I think getattr(self, self.DATA_ATTR_NAME) returns the the whole data structure.  I'm not a hundred percent why, but I think what is going on is self refers back to the data.